### PR TITLE
Creating add an album views

### PR DESCRIPTION
**Pull request by jillianw**

### CODE REVIEW

**Comment by jillianw** [comment]
We can't figure out how to make the add an album/artist page flow to the next page (add artist/band's members) while hanging onto the new artist's id number. Can we get some help with this process?

File: app/controllers/albums_controller.rb [hunk 1]
```diff
@@ -17,12 +17,26 @@ def index
...
     @albums   = Album.all
     @artists  = Artist.all
     @areas    = Area.all
     @genres   = Genre.all
     @formats  = Format.all
-    @releases = Release.all    
+    @releases = Release.all
+  end
+
+  def create
```
Comment (sumeetjain) [inline app/controllers/albums_controller.rb:30]: Has approx 7 statements (TooManyStatements)

File: app/controllers/albums_controller.rb [hunk 2]
```diff
@@ -17,12 +17,26 @@ def index
     @releases = Release.all
   end
 
-  def user_addition
+  def new
```
Comment (sumeetjain) [inline app/controllers/albums_controller.rb:20]: Has approx 6 statements (TooManyStatements)

File: app/controllers/artists_controller.rb [hunk 3]
```diff
@@ -18,4 +18,21 @@ def index
     @areas    = Area.all
     @formats  = Format.all
   end
+
+  def new_members
```
Comment (sumeetjain) [inline app/controllers/artists_controller.rb:23]: Has approx 7 statements (TooManyStatements)

File: app/controllers/albums_controller.rb [hunk 4]
```diff
@@ -17,12 +17,26 @@ def index
...
     @genres   = Genre.all
     @formats  = Format.all
-    @releases = Release.all    
+    @releases = Release.all
+  end
+
+  def create
+    # @album = Album.find_by_id(params[:album_title])
+    @album = Album.new
+    @album.title = (params[:album_title])
```
Comment (sumeetjain) [inline app/controllers/albums_controller.rb:32]: Don't use parentheses around a method call.

File: app/controllers/albums_controller.rb [hunk 5]
```diff
@@ -17,12 +17,26 @@ def index
...
+    @album.save
+
+    @artist = Artist.new
+    @artist.name = (params[:artist_name])
```
Comment (sumeetjain) [inline app/controllers/albums_controller.rb:36]: Don't use parentheses around a method call.

File: app/controllers/albums_controller.rb [hunk 6]
```diff
@@ -17,12 +17,26 @@ def index
...
+    @artist.save
+
+    redirect_to("/artists/#{:id}/new_members")
```
Comment (sumeetjain) [inline app/controllers/albums_controller.rb]: Literal interpolation detected.

File: app/controllers/artists_controller.rb [hunk 7]
```diff
@@ -18,4 +18,21 @@ def index
...
+    #NEED TO REMOVE THESE ONCE SIDEBAR INFO IS EXCLUDED FROM THIS PAGE
```
Comment (sumeetjain) [inline app/controllers/artists_controller.rb]: Missing space after #.

File: app/controllers/artists_controller.rb [hunk 8]
```diff
@@ -18,4 +18,21 @@ def index
...
+    @albums   = Album.all
+    @artists  = Artist.all
+    @areas    = Area.all
+    @genres   = Genre.all
+    @formats  = Format.all
+    @releases = Release.all
+
+    @artist = Artist.find_by_id(params[:artist_name])
+
```
Comment (sumeetjain) [inline app/controllers/artists_controller.rb]: Extra empty line detected at method body end.

File: app/controllers/artists_controller.rb [hunk 9]
```diff
@@ -18,4 +18,21 @@ def index
...
+  end
+
+  def create_members
+
```
Comment (sumeetjain) [inline app/controllers/artists_controller.rb:70]: Extra empty line detected at method body beginning.

**Comment by sumeetjain** [comment]
I'm heading over!

File: app/controllers/artists_controller.rb [hunk 10]
```diff
@@ -19,4 +19,32 @@ def index
...
+  def create_members
+    @member = Member.new
+    @member.name = (params[:member_name])
+    @member.save
+
+    @instruments = Instrument.find_by_id(params[:id])
+    @instruments.name = (params[:instrument_name])
+    @instruments.save
+
+    # Need to modify to NOT add new instruments but rather select from
```
Comment (sumeetjain) [inline app/controllers/artists_controller.rb]: Remove trailing whitespace

File: app/controllers/artists_controller.rb [hunk 11]
```diff
@@ -19,4 +19,32 @@ def index
...
+    @areas    = Area.all
+    @genres   = Genre.all
+    @formats  = Format.all
+    @releases = Release.all
+
+    @artist = Artist.find_by_id(params[:id])
+    @instruments = Instrument.all
+  end
+
+  def create_members
```
Comment (sumeetjain) [inline app/controllers/artists_controller.rb:36]: Has approx 7 statements (TooManyStatements)

File: app/controllers/artists_controller.rb [hunk 12]
```diff
@@ -19,4 +19,32 @@ def index
     @areas    = Area.all
     @formats  = Format.all
   end
+
+  def new_members
```
Comment (sumeetjain) [inline app/controllers/artists_controller.rb:23]: Has approx 8 statements (TooManyStatements)

File: app/controllers/albums_controller.rb [hunk 13]
```diff
@@ -17,12 +17,25 @@ def index
...
     @areas    = Area.all
     @genres   = Genre.all
     @formats  = Format.all
-    @releases = Release.all    
+    @releases = Release.all
+  end
+
+  def create
+    @album = Album.new
+    @album.title = (params[:album_title])
```
Comment (sumeetjain) [inline app/controllers/albums_controller.rb:32]: Don't use parentheses around a method call.

File: app/controllers/artists_controller.rb [hunk 14]
```diff
@@ -19,4 +19,32 @@ def index
...
+    @formats  = Format.all
+    @releases = Release.all
+
+    @artist = Artist.find_by_id(params[:id])
+    @instruments = Instrument.all
+  end
+
+  def create_members
+    @member = Member.new
+    @member.name = (params[:member_name])
```
Comment (sumeetjain) [inline app/controllers/artists_controller.rb]: Don't use parentheses around a method call.

File: app/controllers/artists_controller.rb [hunk 15]
```diff
@@ -19,4 +19,32 @@ def index
...
+    @member.save
+
+    @instruments = Instrument.find_by_id(params[:id])
+    @instruments.name = (params[:instrument_name])
```
Comment (sumeetjain) [inline app/controllers/artists_controller.rb]: Don't use parentheses around a method call.

File: app/controllers/artists_controller.rb (hunk 10 above)
Comment (sumeetjain) [inline app/controllers/artists_controller.rb]: Trailing whitespace detected.

File: app/controllers/artists_controller.rb [hunk 16]
```diff
@@ -19,4 +19,32 @@ def index
...
+    @instruments.save
+
+    # Need to modify to NOT add new instruments but rather select from 
+
+    redirect_to("/artists/#{@artist.id}/new_members")
+
```
Comment (sumeetjain) [inline app/controllers/artists_controller.rb:70]: Extra empty line detected at method body end.

**Pull request merged**